fairroot/parbase/FairParSet.cxx

```diff
@@ -47,30 +47,34 @@ Bool_t FairParSet::init()
     // the second input. If this failes too, it returns an error.
     // (calls internally the init function in the derived class)
     FairRuntimeDb* rtdb = FairRuntimeDb::instance();
-    // FairRunAna* fRun =FairRunAna::Instance();
-    // cout << "-I-  FairParSet::init() " << GetName() << endl;
+    if (!rtdb) {
+        LOG(error) << "FairParSet::init()/" << GetName() << ": No RTDB";
+        return false;
+    }
 
     Bool_t allFound = kFALSE;
-    FairParIo* io = 0;
-    if (rtdb) {
-        io = rtdb->getFirstInput();
+    FairParIo* io = rtdb->getFirstInput();
+    if (io) {
+        allFound = init(io);
+    } else {
+        LOG(warn) << "FairParSet::init()/" << GetName() << ": No first input";
```

Comment:
This warning (just as the second one) is unnecessary and would be too verbose.
The users may (from what I recall) use `First` or `Second` input independently.

Reply:
Okay, I'll turn it into a `debug`, okay?

Reply:
I factored the "which input is there and not" into the FairRuntimeDb::initContainers caller, which now lists the available inputs.